OasisDEX/oasis-borrow
# Oracless positions on owner's page

# [Oracless positions on owner's page](https://app.shortcut.com/oazo-apps/story/10717/oracless-positions-on-owner-s-page)

**Important note**: besides support for oracless pool on owner's page, this PR also contains removal of all non longer needed parts of owner's page, that were used with different feature flags combinations - hence over 1k removed lines.

![image](https://github.com/OasisDEX/oasis-borrow/assets/16230404/69164271-981c-4701-a778-3f0dca81333f)
  
## Changes 👷‍♀️
- Adjusted Ajna observables so they are able to recognize if pool is oracless,
- updated `createReadPositionCreatedEvents$` to use token identifier for supporting oracless pools in DPM data,
- removed `Ajna` and `FollowVault` feature flags from owner's page and all types, helpers, components and observables that were used just by the content when flags were false,
- bonus: fixed annoying bug with missing key in navigation causing constant warning spam in console.
  
## How to test 🧪

Either open your own oracless position and check if it's available on owner's page, or check `0xc4c4576092B77DfE133C9928aaD483271C1226B1` position page on goerli and look for two TCOIN/TDAI positions.
